crates/crypto_api/src/buffer/buffer_vec_u8.rs

```diff
@@ -19,4 +19,21 @@ impl Buffer for Vec<u8> {
     fn set_no_access(&self) {}
     fn set_readable(&self) {}
     fn set_writable(&self) {}
+
+    fn compare(&mut self, b: &mut Box<dyn Buffer>) -> i32 {
```

Comment:
what do you think about doing this as a provided implementation in the Trait, like https://github.com/holochain/lib3h/blob/develop/crates/crypto_api/src/buffer/mod.rs#L59-L62 ?

Reply:
done